testnet/local/README.md

```diff
@@ -0,0 +1,87 @@
+# Docker
+This docker-compose configuration is meant to be used in combination with the [Meson server](https://github.com/hashcloak/Meson-server/tree/develop) and [Katzenmint PKI](https://github.com/hashcloak/katzenmint-pki/tree/develop) repositories. It is meant for testing client and server mix network components as part of the core Meson developer work flow. It should be obvious that this docker-compose situation is not meant for production use.
+
+1. clone meson server and build docker image of meson server
+```BASH
+$ git clone https://github.com/hashcloak/Meson-server.git
+$ cd Meson-server
+$ docker build --no-cache -t meson/server .
+```
+
+2. build docker image for katzenmint-pki
+```BASH
+$ cd katzenmint-pki
+$ docker build --no-cache -t katzenmint/pki .
+```
+
+3. start three katzenmint pki nodes / three mix nodes / two providers.
+```BASH
+$ docker-compose up
+```
+
+4. checkout information of katzenmint pki nodes with curl command
+```BASH
+# node1
+$ curl http://127.0.0.1:21483/net_info
+
+# node2
+$ curl http://127.0.0.1:21484/net_info
+
+# node3
+$ curl http://127.0.0.1:21485/net_info
+```
+
+# Examples
+
+## Ping
+[Ping](https://github.com/sc0Vu/mixnet-examples/tree/master/ping) is a mixnet tool for testing and debugging. After start local mixnets, you can test with ping tool.
+
+1. clone
+```BASH
+$ git clone https://github.com/sc0Vu/mixnet-examples.git
+$ cd mixnet-examples/ping
+```
+
+2. update `katzenpost.toml
```

Comment:
and simply use the example toml file in Meson/ping/

Reply:
Thanks, updated!